contact.html

```diff
@@ -31,17 +31,19 @@
       <p>Contact me at <span class="email"> michael.dav.kern (at) gmail.com </span>, or drop me a line here:</p>
 
     <form action="form.html">
-    <label>Name</label>
-    <input type="text" name="name" required />
-    <label>Email</label>
+    <label>Name:</label>
+    <input type="text" name="name" required /></br>
+    <label>Email:</label>
     <input type="email" name="email" required /></br>
-    <select>
+    <select class="dropDown">
       <option value="pro">Designated hitters are awesome</option>
-      <option value="against">No way! Designated hitters are lame</option>
+      <option value="against">Designated hitters are lame</option>
     </select></br>
-    How can I help you?<input type="text" name="note" required /></br>
+    How can I help you?<input class="submissionField" type="text" name="note" required /></br>
```

Comment:
Look into the `<textarea>` tag instead of styling an `<input>` to make it taller.